Tools/CISupport/Shared/generate-s3-url

```diff
@@ -16,12 +16,15 @@ if 'uat' in hostname:
 S3_DEFAULT_BUCKET = f'archives.webkit{custom_suffix}.org'
 S3_EWS_BUCKET = f'ews-archives.webkit{custom_suffix}.org'
 
-def generateS3URL(bucket, identifier, revision):
+def generateS3URL(bucket, identifier, revision, extension=None, content_type=None):
```

Comment:
can have extension='zip'

Reply:
We can't do that here because `--extension` has a default of `None`, so we need the default to be set in the function itself.